assets/queries/k8s/cap_drop_any_capabilities/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "cap_drop_any_capabilities",
+  "queryName": "Cap Drop Any Capabilities",
+  "severity": "LOW",
+  "category": null,
+  "descriptionText": "Sees if Kubernetes Drop exists",
+  "descriptionUrl": "https://kubesec.io/basics/containers-securitycontext-capabilities-drop-index-all/"
```

Comment:
Query Name could be improved and description is too vague. Also, descriptionUrl should point to official kubernetes documentation